profile.go

```diff
@@ -130,7 +130,7 @@ This profile may only be applied when first initializing the node.
 This is the most battle-tested and reliable datastore, but it's significantly
 slower than the badger datastore. You should use this datastore if:
 
-* You need a very simple and very reliable datastore you and trust your
+* You need a very simple and very reliable datastore and you trust your
```

Comment:
nit: while a bit silly this might be better off in it's own `chore: fix doc typo` commit

Reply:
@aschmahmann Agree this sort of thing should normally be a separate PR, but thought this ok as an exception, since I already needed to edit other lines in the badger1 description.